Appineers_V4/application/webservice/wsengine/controllers/Wscontroller.php

```diff
@@ -101,7 +101,7 @@ public function WSExecuter($func_arg = '', $id_arg = null)
             $msg = "App maintenance mode";
         }
 
-       $api_allowed_without_access_token = array('social_login','user_login_phone','user_login_email','send_verification_link','check_unique_user','social_sign_up','user_sign_up_phone','user_sign_up_email','user_email_confirmation','static_pages','send_sms','reset_password_confirmation','reset_password_phone','get_template_message','forgot_password_phone','reset_password','forgot_password','states_list','get_config_paramaters','category_list','category_with_sub');
+       $api_allowed_without_access_token = array('social_login','user_login_phone','user_login_email','send_verification_link','check_unique_user','social_sign_up','user_sign_up_phone','user_sign_up_email','user_email_confirmation','static_pages','send_sms','reset_password_confirmation','reset_password_phone','get_template_message','forgot_password_phone','reset_password','forgot_password','states_list','get_config_paramaters','category_list','category_with_sub','demo_api');
```

Comment:
$api_allowed_without_access_token = array('social_login','user_login_phone','user_login_email','send_verification_link','check_unique_user','social_sign_up','user_sign_up_phone','user_sign_up_email','user_email_confirmation','static_pages','send_sms','reset_password_confirmation','reset_password_phone','get_template_message','forgot_password_phone','reset_password','forgot_password','states_list','get_config_paramaters','category_list','category_with_sub','demo_api');

why added this?

Reply:
controller method should be called from  without authorisation token from post man